core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -819,10 +819,6 @@ public static FunctionExpression castTimestamp(Expression value) {
     return compile(FunctionProperties.None, BuiltinFunctionName.CAST_TO_TIMESTAMP, value);
   }
 
-  public static FunctionExpression castDatetime(Expression value) {
```

Comment:
I believe this would be as there is no such thing as a datetime value to cast to anymore, unless it returns a timestamp which sounds unreasonable as we already have the castTimestamp to do that